package.json

```diff
@@ -36,12 +36,16 @@
     }
   },
   "devDependencies": {
-    "@storybook/react": "^6.5.16",
+    "@gravitational/build": "workspace:*",
```

Comment:
I'm not sure if this needs to be here. 🤔

Reply:
the vite config filein storybook wasn't able to find this package without having this here. i could use relative paths but i wanted to keep the import cleaner